agenthub/self_discover_agent/reasoning_action.py

```diff
@@ -0,0 +1,119 @@
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, TypeGuard, Union
+
+from .agent_state_machine import SelfDiscoverState
+
+# Order is important as it is used in prompt
+RESEASONING_MODULE_LIST: List[str] = [
```

Comment:
```suggestion
REASONING_MODULE_LIST: List[str] = [
```
Just some nitpick :)

Reply:
Fixed :)